cmd/skupper/skupper.go

```diff
@@ -416,6 +416,12 @@ func main() {
 				if vir.Status.TransportReadyReplicas == 0 {
 					fmt.Printf("Skupper is enabled for namespace %q%s. Status pending...", cli.Namespace, modedesc)
 				} else {
+					if len(vir.Status.ConnectedSites.Warnings) > 0 {
```

Comment:
I think this if is not needed, if "len=0" it just do not iterate

Reply:
The len check will also return 0 if the slice is nil. Arguably you could just check if it is nil or not here, but to me the len is a nice way of checking whether iteration is needed in either case.

Reply:
aha!